e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/deal-cancellation-enabled/dashboard-deals-filters.spec.js

```diff
@@ -0,0 +1,25 @@
+const { DEAL_STATUS } = require('@ukef/dtfs2-common');
+const MOCK_USERS = require('../../../../../../../e2e-fixtures');
+const { dashboardDeals } = require('../../../../pages');
+const { dashboardFilters } = require('../../../../partials');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+const filters = dashboardFilters;
+
+context('Dashboard Deals filters - Deal cancellation enabled', () => {
+  before(() => {
+    cy.login(BANK1_MAKER1);
+  });
+
+  describe('When deal cancellation is enabled on the dashboard deals page', () => {
+    it('displays the deal cancellation status filter', () => {
+      dashboardDeals.visit();
+      filters.showHideButton().click();
+
+      dashboardDeals.filters.panel.form.status.cancelled.label().contains(DEAL_STATUS.CANCELLED);
```

Comment:
Are we able to use `cy.assertText` here instead of `.contains`?

Reply:
Good spot! have changed, thanks :)